### ISSUE TITLE
Is a Windows port possible? Or Android?

### ISSUE THREAD

**Issue by John-Enigma** [washingtondc-emu/washingtondc#49]
I ask considering this emulator is restricted to Linux only.

**Comment by snickerbockers** [comment]
Yes, there's definitely going to be a Windows port in the near future.  The
reason why it's still linux-exclusive for now is that I make large
refactors sometimes, and I don't want to do that across two different
platforms until I've gotten things stable.

The Android port is less likely to happen any time soon because
WashingtonDC unconditionally assumes that the host architecture is
Little-Endian, but Android is Big-Endian and also because Android isn't a
platform that I spend much time using.
